shell/lib/asdf.sh

```diff
@@ -40,9 +40,24 @@ asdf_devbase_exec() {
   # shellcheck disable=SC2155
   local tool_env_var="$(echo "$tool" | tr '[:lower:]-' '[:upper:]_')"
 
-  # Why: We're OK with this being the way it is.
-  # shellcheck disable=SC2155
-  local version="$(asdf_get_version_from_devbase "$tool")"
+  # Allow overriding the golangci-lint version with an environmental variable
+  if [[ $tool = "golangci-lint" ]] && [[ "${GOLANGCI_LINT_VER}" ]]; then
```

Comment:
Putting a version in the top block in the `.tool-versions` should already be overriding... so I'm not really sure this is required.

Reply:
Talking more offline -- realized that this _is_ required for devbase managed versions. We need the ability for the `.tool-versions` in the root of the repo to override the devbase specified version, which isn't a thing today.